tony-core/src/main/java/com/linkedin/tony/util/Utils.java

```diff
@@ -618,5 +618,50 @@ public static void extractResources() {
     }
   }
 
+  /**
+   * Parses Docker related configs and sets the appropriate container environment variables if Docker is available.
+   * Uses reflection to support older versions of Hadoop.
+   */
+  public static boolean parseDockerConfigs(Configuration tonyConf, Map<String, String> containerEnv, String jobType) {
+    if (tonyConf.getBoolean(TonyConfigurationKeys.DOCKER_ENABLED, TonyConfigurationKeys.DEFAULT_DOCKER_ENABLED)) {
+      String imagePath = tonyConf.get(TonyConfigurationKeys.getContainerDockerKey());
+      if (tonyConf.get(TonyConfigurationKeys.getDockerImageKey(jobType)) != null) {
+        imagePath = tonyConf.get(TonyConfigurationKeys.getDockerImageKey(jobType));
```

Comment:
Use local variable for `tonyConf.get(TonyConfigurationKeys.getDockerImageKey(jobType))` instead of repeating.